cohorts/views.py

```diff
@@ -2409,14 +2412,16 @@ def cohort_files(request, cohort_id, limit=20, page=1, offset=0, build='HG38', a
                         'project_short_name': (item['project_short_name'] or 'N/A'),
                         'cohort_id': cohort_id
                     })
-
-            if not files_only:
-                # Add up the file total
-                for attr in filter_counts:
-                    for val in filter_counts[attr]:
-                        if attr not in inc_filters or val in inc_filters[attr]:
-                            total_file_count += int(filter_counts[attr][val])
-
+            filter_counts = counts
+            files_counted = False
+            # Add to the file total
+            for attr in filter_counts:
```

Comment:
This section can't be run if there's nothing to count, so it needs to be re-indented and put behind if do_file_counts: